Romina.Api/Repositories/ProductRepository.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Generic;
+using Dapper;
+using Romina.Api.Models;
+using Romina.Api.Settings;
+using System.Data.SqlClient;
+using System.Linq;
+
+namespace Romina.Api.Repositories
+{
+    public class ProductRepository: IProductRepository
+    {
+        private readonly SqlSettings settings;
+
+        public ProductRepository(SqlSettings settings)
+        {
+            this.settings = settings;
+        }
+
+        public Product GetProductById(string id)
+        {
+            Product product = null;
+            using (SqlConnection connection = new SqlConnection(settings.ConnectionString))
+            {
+                var products = connection.Query<Product>("SELECT * FROM PRODUCT WHERE ID= @id",id)
+                    .ToList();
+
+                product = products[0];
+            }
+
+            return product;
+        }
+
+        public Product GetProductByMake(string basicMake)
```

Comment:
we do need the ability both get a product by its make or model, but we can probably clean this up to not have multiple methods 

Look at `feature/dans-reference` to see what I mean